lib/resolvers.js

```diff
@@ -141,7 +141,9 @@ const createProxyResolvers = function (component, resolvers) {
         proxyResolvers[rootType][field] = resolver;
         continue;
       }
-      proxyResolvers[rootType][field] = createProxyResolver(component, rootType, field);
+      proxyResolvers[rootType][field] = rootType === 'Subscription' ? {
```

Comment:
probably more readable in this way:
proxyResolvers[rootType][field] = (rootType === 'Subscription') ? {

Reply:
sounds good I'll make that change